store/tikv/async_commit_test.go

```diff
@@ -266,6 +282,79 @@ func (s *testAsyncCommitSuite) TestCheckSecondaries(c *C) {
 	c.Assert(gotOther, Equals, int64(0))
 }
 
+func (s *testAsyncCommitSuite) TestRepeatableRead(c *C) {
+	config.UpdateGlobal(func(conf *config.Config) {
+		conf.TiKVClient.EnableAsyncCommit = true
+	})
+	defer config.RestoreFunc()()
+
+	test := func(isPessimistic bool) {
+		s.putKV(c, []byte("k1"), []byte("v1"))
+
+		ctx := context.Background()
+		txn1, err := s.store.Begin()
+		txn1.SetOption(kv.Pessimistic, isPessimistic)
+		c.Assert(err, IsNil)
+		s.mustGetFromTxn(c, txn1, []byte("k1"), []byte("v1"))
+		txn1.Set([]byte("k1"), []byte("v2"))
+
+		for i := 0; i < 20; i++ {
+			_, err := s.store.GetOracle().GetTimestamp(ctx)
+			c.Assert(err, IsNil)
+		}
+
+		txn2, err := s.store.Begin()
+		c.Assert(err, IsNil)
+		s.mustGetFromTxn(c, txn2, []byte("k1"), []byte("v1"))
+
+		err = txn1.Commit(ctx)
+		c.Assert(err, IsNil)
+		s.mustGetFromTxn(c, txn2, []byte("k1"), []byte("v1"))
+		err = txn2.Rollback()
+		c.Assert(err, IsNil)
+
+		txn3, err := s.store.Begin()
+		c.Assert(err, IsNil)
+		s.mustGetFromTxn(c, txn3, []byte("k1"), []byte("v2"))
+		err = txn3.Rollback()
+		c.Assert(err, IsNil)
+	}
+
+	test(false)
+	test(true)
+}
+
+func (s *testAsyncCommitSuite) TestPointGetWithAsyncCommit(c *C) {
+	config.UpdateGlobal(func(conf *config.Config) {
+		conf.TiKVClient.EnableAsyncCommit = true
+	})
+	defer config.RestoreFunc()()
+
+	s.putAlphabets(c)
+
+	txn, err := s.store.Begin()
+	c.Assert(err, IsNil)
+	txn.Set([]byte("a"), []byte("v1"))
+	txn.Set([]byte("b"), []byte("v2"))
+	s.mustPointGet(c, []byte("a"), []byte("a"))
+	s.mustPointGet(c, []byte("b"), []byte("b"))
+
+	// PointGet cannot ignore async commit transactions' locks.
+	c.Assert(failpoint.Enable("github.com/pingcap/tidb/store/tikv/asyncCommitDoNothing", "return"), IsNil)
```

Comment:
Need we disable this failpoint after this test?

Reply:
Oh sorry I forgot it

Reply:
I think that was the problem that caused CI fail..